fbw-a380x/src/systems/instruments/src/MFD/pages/common/Button.tsx

```diff
@@ -0,0 +1,256 @@
+import {
+  ComponentProps,
+  DisplayComponent,
+  FSComponent,
+  Subject,
+  Subscribable,
+  Subscription,
+  VNode,
+} from '@microsoft/msfs-sdk';
+import './style.scss';
+import { TriangleDown, TriangleUp } from 'instruments/src/MFD/pages/common/shapes';
+
+export type ButtonMenuItem = {
+  label: string;
+  action(): void;
+};
+
+export interface ButtonProps extends ComponentProps {
+  label: string | Subscribable<VNode>;
+  menuItems?: Subscribable<ButtonMenuItem[]>; // When defining menu items, idPrefix has to be set
+  showArrow?: boolean;
+  idPrefix?: string;
+  disabled?: Subscribable<boolean>;
+  selected?: Subscribable<boolean>; // Renders with lighter grey if selected (e.g. for segmented controls)
+  buttonStyle?: string;
```

Comment:
We also probably shouldn't allow an explicit style property to be passed in here. Use a CSS class instead.

Reply:
If you look at where it's used, that would be about ~30 new CSS classes with a single use. I'd strongly advise against it